### DISCUSSION TITLE
Conformance Language

### DISCUSSION THREAD

**Discussion by shamilnn** [fhir-ru/core#51] [General]
Не нашёл перевода языка соответствия на русский язык - https://fhir-ru.github.io/conformance-rules.html#conflang

Следующий параграф должен присутствовать где-то в RU Core IG и conformance words/verbs использоваться во всех текстах для однообразия (чтобы исключить  путаницы во всех этих "обязан", "должен", "может", "возможно" и т.д., и т.п.) 
Вероятно, это можно взять из требований к CDA.

_This specification uses the conformance verbs SHALL, SHOULD, and MAY as defined in RFC 2119. Unlike RFC 2119, however, this specification allows that different applications might not be able to interoperate because of how they use optional features. In particular:
    **SHALL**: an absolute requirement for all implementations
    **SHALL NOT**: an absolute prohibition against inclusion for all implementations
    **SHOULD/SHOULD NOT**: A best practice or recommendation to be considered by implementers within the context of their particular implementation; there may be valid reasons to ignore an item, but the full implications must be understood and carefully weighed before choosing a different course
    **MAY**: This is truly optional language for an implementation; can be included or omitted as the implementer decides with no implications_

**Comment by iradio** [comment]
Полагаю стоит договориться и о однозначном переводе определений. Предлагаю зафиксировать так:
ОБЯЗАН: an absolute requirement for all implementations
ДОЛЖЕН: A best practice or recommendation to be considered by implementers within the context of their particular implementation; there may be valid reasons to ignore an item, but the full implications must be understood and carefully weighed before choosing a different course
МОЖЕТ: This is truly optional language for an implementation; can be included or omitted as the implementer decides with no implications
